paasta_tools/api/views/instance.py

```diff
@@ -845,3 +845,60 @@ def get_deployment_version(
 ) -> Optional[str]:
     key = ".".join((cluster, instance))
     return actual_deployments[key][:8] if key in actual_deployments else None
+
+
+@view_config(
+    route_name="service.instance.mesh_status", request_method="GET", renderer="json",
+)
+def instance_mesh(request):
+    service = request.swagger_data.get("service")
+    instance = request.swagger_data.get("instance")
+    verbose = request.swagger_data.get("verbose")
+    include_smartstack = request.swagger_data.get("include_smartstack")
+    include_envoy = request.swagger_data.get("include_envoy")
+
+    instance_mesh: Dict[str, Any] = {}
+    instance_mesh["service"] = service
+    instance_mesh["instance"] = instance
+
+    try:
+        instance_type = validate_service_instance(
+            service, instance, settings.cluster, settings.soa_dir
+        )
+    except NoConfigurationForServiceError:
+        error_message = (
+            f"Deployment key {settings.cluster}.{instance} not found. "
```

Comment:
Is this the right error message? I think the above function validates if the instance name exists in soaconfigs.

Reply:
It's the same error message for all the other times we validate the instance name in other routes.

Reply:
Yeah, but I think it may have been incorrect that time, too... Can you double-check that the message describes the error correctly?

Reply:
I'm mostly concerned about making sure that the tip leads down the right path.

Reply:
ive changed it to "No instance {service}.{instance} is configured to run in the {cluster} cluster", which should make more sense.